__init__.py

```diff
@@ -428,12 +504,18 @@ def register():
     bpy.types.Scene.saveTextures = bpy.props.BoolProperty(name="Save Textures As PNGs (Breaks CI Textures)")
     bpy.types.Scene.generateF3DNodeGraph = bpy.props.BoolProperty(name="Generate F3D Node Graph", default=True)
     bpy.types.Scene.exportHiddenGeometry = bpy.props.BoolProperty(name="Export Hidden Geometry", default=True)
-    bpy.types.Scene.blenderF3DScale = bpy.props.FloatProperty(name="F3D Blender Scale", default=100)
+    bpy.types.Scene.blenderF3DScale = bpy.props.FloatProperty(
+        name="F3D Blender Scale", default=100, update=on_update_render_settings
+    )
 
     bpy.types.Scene.fast64 = bpy.props.PointerProperty(type=Fast64_Properties, name="Fast64 Properties")
     bpy.types.Bone.fast64 = bpy.props.PointerProperty(type=Fast64_BoneProperties, name="Fast64 Bone Properties")
     bpy.types.Object.fast64 = bpy.props.PointerProperty(type=Fast64_ObjectProperties, name="Fast64 Object Properties")
 
+    bpy.types.Scene.alreadyLinkedMaterialNodes = bpy.props.BoolProperty(
```

Comment:
if truly needed can this go into Scene.fast64
what is this for?

Reply:
this is me forgetting to clean up something that I ended up not using 😔

Reply:
done https://github.com/Fast-64/fast64/pull/109/commits/ef56bb1ecbf0d04b48ea3093755d558e76f7618c